test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -193,15 +193,15 @@ public class ValueTypeTests {
 	static MethodHandle getUnAlignedObjectflatObjectBackfill2InstanceL = null;
 
 	/* fields */
-	static String typeWithSingleAlignmentFields[] = {
+	static String[] typeWithSingleAlignmentFields = {
```

Comment:
While we're tidying up, can you remove the use of raw types too, please (e.g. Class<?> instead of just Class)?

Reply:
Raw types are still used on lines 2526 and 2628.